### DISCUSSION TITLE
OpenSlides 4.0 on Port 443

### DISCUSSION THREAD

**Discussion by aabeinsst** [OpenSlides/OpenSlides#6621] [Q&A]
Hi,
i want to make OpenSlides accessable on port 443 of my domain. When configuring with this port the server gets accessable under this port but it only shows the login page without the form (so only the header-logo).
The browser-console shows:

> operator: user changed from  -1 to null main.97ceee9396c51980.js:1:579084
operator: active meeting changed from  undefined to null main.97ceee9396c51980.js:1:579629
apps loaded! main.97ceee9396c51980.js:1:566149
App is stable now. main.97ceee9396c51980.js:1:811869
auth: Do WhoAmI main.97ceee9396c51980.js:1:538107
Das Cookie "refreshId" verfügt über keinen gültigen Wert für das "SameSite"-Attribut. Bald werden Cookies ohne das "SameSite"-Attribut oder mit einem ungültigen Wert dafür als "Lax" behandelt. Dadurch wird das Cookie nicht länger an Kontexte gesendet, die zu einem Drittanbieter gehören. Falls Ihre Anwendung das Cookie in diesen Kontexten benötigt, fügen Sie bitte das Attribut "SameSite=None" zu ihm hinzu. Weitere Informationen zum "SameSite"-Attribut finden Sie unter https://developer.mozilla.org/docs/Web/HTTP/Headers/Set-Cookie/SameSite. 2 who-am-i
auth: WhoAmI done, online: true authenticated: false main.97ceee9396c51980.js:1:538240
Lifecycle: shutdown. main.97ceee9396c51980.js:1:565272
Lifecycle: booted. main.97ceee9396c51980.js:1:565153
[autoupdate] new request: organization_detail:subscription 
Array [ {…}, {…} ]
main.97ceee9396c51980.js:1:548139
has meeting changed? 
Object { currentMeetingId: undefined, nextMeetingId: null, hasChanged: true }
main.97ceee9396c51980.js:1:574918
undefined polyfills.9e5f7f0827d1b4c1.js:1:27901
Object { _zone: {…}, runCount: 0, _zoneDelegates: [], _state: "scheduling", type: "macroTask", source: "setTimeout", data: {…}, scheduleFn: d(v), cancelFn: w(v), callback: 0(), … }
polyfills.9e5f7f0827d1b4c1.js:1:27953
undefined polyfills.9e5f7f0827d1b4c1.js:1:27901
true false null main.97ceee9396c51980.js:1:180672
undefined polyfills.9e5f7f0827d1b4c1.js:1:27901
Object { _zone: {…}, runCount: 0, _zoneDelegates: [], _state: "scheduling", type: "macroTask", source: "setTimeout", data: {…}, scheduleFn: d(v), cancelFn: w(v), callback: 0(), … }
polyfills.9e5f7f0827d1b4c1.js:1:27953

Did I miss some configurations to make this possible?
Hope you can help me!

**Comment by jsangmeister** [comment]
Sorry for the missing answer. If the problem still persists, feel free to open a new discussion/issue and we will try to help you. I'm closing this for now.
